cli/plasmo/src/features/manifest-factory/base.ts

```diff
@@ -224,6 +224,19 @@ export abstract class PlasmoManifest<T extends ExtensionManifest = any> {
         }
       })
     )
+
+    if (!process.env.POST_BUILD_SCRIPT)
+      return
+
+    const postBuildPath = resolve(process.env.POST_BUILD_SCRIPT)
+
+    if (!existsSync(postBuildPath)) {
+      console.error("Post-build file does not exist or is not readable:", postBuildPath)
+      return
+    }
+
+    const postBuild = require(postBuildPath)
```

Comment:
nit: would be nice if this script could also be in TS 🤑 (maybe run it via tsx, or run it via some type of bundling technique :-?)

Reply:
ref: https://github.com/PlasmoHQ/plasmo/issues/993